ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/ExtraConfig.scala

```diff
@@ -6,12 +6,14 @@ package com.daml.ledger.on.memory
 import com.daml.ledger.participant.state.kvutils.api.BatchingLedgerWriterConfig
 
 final case class ExtraConfig(
-    batchingLedgerWriterConfig: BatchingLedgerWriterConfig
+    batchingLedgerWriterConfig: BatchingLedgerWriterConfig,
+    preExecute: Boolean,
```

Comment:
Because ideally we'd be deciding whether to pre-execute a transaction based on its interpretation cost I'd suggest naming this configuration parameter as `alwaysPreExecute` instead (and its associated command-line argument similarly as well).

Reply:
Done.